src/types/RickApiResponse.ts

```diff
@@ -0,0 +1,9 @@
+export interface RickApiResponse<T> {
```

Comment:
I would change the name to a more generic one, like ApiResponse. 
We added the Generic Type to be able to pass the structure of the results to the app, but the app shouldn't know the implementation details.
What do you think?